.travis.yml

```diff
@@ -4,7 +4,7 @@ language: python
 
 matrix:
   include:
-    - python: 2.7
+#    - python: 2.7
```

Comment:
I guess it was since travis is complaining. It seems that [this](https://github.com/mrocklin/streams/pull/32/files#diff-14d53a4d1028cb6ff92ab388f7274169R533) is not valid syntax.